go/protocol/keybase1/extras.go

```diff
@@ -2324,10 +2330,18 @@ func (r TeamRole) IsReaderOrAbove() bool {
 	return r.IsOrAbove(TeamRole_READER)
 }
 
+func (r TeamRole) IsBotOrAbove() bool {
+	return r.IsOrAbove(TeamRole_BOT)
+}
+
 func (r TeamRole) IsRestrictedBotOrAbove() bool {
 	return r.IsOrAbove(TeamRole_RESTRICTEDBOT)
 }
 
+func (r TeamRole) IsBot() bool {
```

Comment:
This name could be easily misunderstood as `r == BOT || r == RESTRICTEDBOT`. Might be less trouble to use `==` at the callsite.

(Or names `IsBotOfAnySort` `IsExactlyBot`)

Reply:
Oh the only place we use it is where we want `BotLike` so i'll convert to that